src/commands/fun/say.shaii.ts

```diff
@@ -10,6 +10,7 @@ export default defineCommand({
   requiresProcessing: false,
   execute: async (message) => {
     if (message.args.length === 0) return `what do you want to say ${randomChoice(SLURS)}`;
+	await message.delete();

```

Comment:
## Don't await the delete 
We do not care if the message is deleted or not and we don't wanna block the thread till this deletes
```ts
message.delete().catch(() => {})
```

Reply:
I changed it, but that means we have to find a solution to make Shaii sending the result without replying to the deleted message